caddyfile/parse.go

```diff
@@ -95,6 +96,25 @@ func (p *parser) begin() error {
 		return nil
 	}
 
+	if ok, name := p.isMacro(); ok {
+		if p.definedMacros == nil {
+			p.definedMacros = map[string][]Token{}
+		}
+		if p.definedMacros[name] != nil {
+			p.Errf("redeclaration of previously declared macro %s", name)
```

Comment:
This error doesn't seem to be triggered!

I declared two blocks directly after each other and both were parsed, but the second one applied with no error raised.